frontend/src/app/lib/InstallManager.js

```diff
@@ -15,13 +15,29 @@ if (typeof navigator !== 'undefined') {
   mam = navigator.mozAddonManager;
 }
 
-function mozAddonManagerInstall(url) {
+function mozAddonManagerInstall(url, sendToGA) {
   return mam.createInstall({ url }).then(install => {
     return new Promise((resolve, reject) => {
+      const start = url.indexOf('files/') + 6;
+      const end = url.indexOf('@');
+      const experimentTitle = url.substring(start, end);
       install.addEventListener('onInstallEnded', () => resolve());
```

Comment:
These three lines can be removed as per https://github.com/mozilla/testpilot/issues/2391